app/controllers/home_controller.rb

```diff
@@ -8,8 +8,15 @@ def index
   end
 
   def dashboard
+    # creates an instance variable called user_resumes
+    # binding.pry
```

Comment:
we can get rid of these 2 comments as they're not needed.

Reply:
deleted those lines in newly committed changes.